<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TalesQtMainWindow</class>
 <widget class="QMainWindow" name="TalesQtMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1059</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tales</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/resources/talesicon.png</normaloff>:/resources/talesicon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="EditorSelectBar" name="selectorBar" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>32</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="MainEditorWidgets" name="editorWidgets" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1059</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad_project"/>
    <addaction name="actionSave_project"/>
    <addaction name="actionSave_project_as"/>
    <addaction name="actionExport_ROM_2"/>
    <addaction name="actionExport_ROM"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionLevel_Editor"/>
    <addaction name="actionVRAM_Editor"/>
    <addaction name="actionGraphics_mappings_editor"/>
    <addaction name="actionPalette_editor"/>
    <addaction name="actionPalette_cycle_editor"/>
    <addaction name="actionMetatile_structure_editor"/>
    <addaction name="actionMetatile_Behavior"/>
    <addaction name="actionLength_map_editor"/>
    <addaction name="actionMusic_editor"/>
    <addaction name="actionPower_up_editor"/>
    <addaction name="actionHacks_and_enhancements"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSave_project">
   <property name="text">
    <string>Save project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionLoad_project">
   <property name="text">
    <string>Open project...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionExport_ROM">
   <property name="text">
    <string>Export ROM as...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+E</string>
   </property>
  </action>
  <action name="actionSave_project_as">
   <property name="text">
    <string>Save project as...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionExport_ROM_2">
   <property name="text">
    <string>Export ROM</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionLevel_Editor">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Level editor</string>
   </property>
   <property name="shortcut">
    <string>Shift+L</string>
   </property>
  </action>
  <action name="actionVRAM_Editor">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>VRAM editor</string>
   </property>
   <property name="shortcut">
    <string>Shift+V</string>
   </property>
  </action>
  <action name="actionMetatile_Behavior">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Metatile behavior editor</string>
   </property>
   <property name="shortcut">
    <string>Shift+B</string>
   </property>
  </action>
  <action name="actionLength_map_editor">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Width/height map editor</string>
   </property>
   <property name="shortcut">
    <string>Shift+M</string>
   </property>
  </action>
  <action name="actionPalette_editor">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Palette editor</string>
   </property>
   <property name="shortcut">
    <string>Shift+P</string>
   </property>
  </action>
  <action name="actionMetatile_structure_editor">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Metatile structure editor</string>
   </property>
   <property name="shortcut">
    <string>Shift+S</string>
   </property>
  </action>
  <action name="actionGraphics_mappings_editor">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Graphics/collision editor</string>
   </property>
   <property name="shortcut">
    <string>Shift+G</string>
   </property>
  </action>
  <action name="actionPower_up_editor">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Power-up editor</string>
   </property>
   <property name="shortcut">
    <string>Shift+R</string>
   </property>
  </action>
  <action name="actionMusic_editor">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Music editor</string>
   </property>
   <property name="shortcut">
    <string>Shift+U</string>
   </property>
  </action>
  <action name="actionPalette_cycle_editor">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Palette cycle editor</string>
   </property>
   <property name="shortcut">
    <string>Shift+Y</string>
   </property>
  </action>
  <action name="actionHacks_and_enhancements">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Hacks and enhancements</string>
   </property>
   <property name="shortcut">
    <string>Shift+H</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>EditorSelectBar</class>
   <extends>QWidget</extends>
   <header>editorselectbar.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MainEditorWidgets</class>
   <extends>QWidget</extends>
   <header>maineditorwidgets.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
